src/components/Scaffolding/Scaffolding.jsx

```diff
@@ -0,0 +1,71 @@
+
+import React from 'react';
+
+import './Scaffolding.scss';
```

Comment:
Can we move local relative path imports to after node_modules imports?

Reply:
Fixed